src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData-CSharp.AppHost/Properties/launchSettings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "https": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "applicationUrl": "https://localhost:17000;http://localhost:15000",
```

Comment:
Should we have the random port generation stuff in here, equivalent to the [`kestrelHttpPortReplacer` from the ASP.NET Core templates](https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/Web.ProjectTemplates/content/EmptyWeb-CSharp/.template.config/template.json#L71)?

I'm not sure if there's an equivalent to that for the other two port numbers below.

Reply:
Thanks - just updated this template to do the same thing that the [Aspire templates](https://github.com/dotnet/aspire/blob/84d6e25c672f9d31bc09ee85102e29d53b0fe85e/src/Aspire.ProjectTemplates/templates/aspire-apphost/.template.config/template.json) do.

Reply:
(i.e., all the ports in this file now get randomly generated)